idl/github.com/uber/cadence/shared.thrift

```diff
@@ -907,7 +913,14 @@ struct DescribeWorkflowExecutionRequest {
   20: optional WorkflowExecution execution
 }
 
+struct PendingActivityInfo {
```

Comment:
@samarabbas do we want to include ActivityType here? If user does not use customized activityID, then the auto assigned ones will be sequential IDs. It could be useful to include those activity type. However, activityType is not part of mutable state (ActivityInfo does not have that data). So if we want to include it here, we would need to update schema to include the data in mutable state.

Reply:
I think including ActivityType here makes a lot of sense.  ActivityInfo has the entire ScheduledEvent, which has ActivityType along with other information.